tools/monorepo-merge/package.json

```diff
@@ -16,6 +16,13 @@
 		"/npm-shrinkwrap.json",
 		"/oclif.manifest.json"
```

Comment:
hmm what is going on in this package. manifest is listed in files, but seems to be rm'd in postpack, and I see you removed the gitignore, is that correct to do? the file is referenced all over the place as if it might exist after a build, so just checking whats the reason to remove the gitignore.

Reply:
The commit that removed them is _super_ old at this point so I don't really remember, sorry! The files in this package were automatically generated with the oclif package, I believe. I can take a look at the ignore file that was removed!